src/ray/raylet/monitor.cc

```diff
@@ -60,6 +60,19 @@ void Monitor::Tick() {
           }
           if (!marked) {
             RAY_CHECK_OK(gcs_client_.client_table().MarkDisconnected(client_id));
+            // Remove all resources of this client from GCS
```

Comment:
Maybe sometimes we'll want query the resources of a dead node. I think we can keep it. it won't waste too much memory.